trac-env/templates/django_theme.html

```diff
@@ -76,6 +76,11 @@ <h1>Issues</h1>
       </div>
     </div>
 
+
+    # if req.authname == 'anonymous' or req.authname == '':
+    <div id="guest"></div>
```

Comment:
What's the advantage of using this empty element rather than for example a class on the `<body>`?